src/forms/ColorMapInput.tsx

```diff
@@ -7,18 +7,18 @@ import SelectBox, { choicesT } from "./SelectBox";
 
 import { validatorResult } from "./validators";
 
-import styles from "./ColorMapInput.css";
-import formStyles from "../styles/Forms.css";
-import inputStyles from "../styles/Input.css";
+import styles from "./ColorMapInput.module.css";
+import formStyles from "../styles/Forms.module.css";
+import inputStyles from "../styles/Input.module.css";
 import CheckBox from "./CheckBox";
 
 import {
   calculateNewStyleAndOptions,
   optionsHasLayers,
-  getColorMapFromStyle,
-  getColorMinFromStyle,
-  getColorMaxFromStyle,
-  getStyleFromOptions,
+  // getColorMapFromStyle,
+  // getColorMinFromStyle,
+  // getColorMaxFromStyle,
+  // getStyleFromOptions,
```

Comment:
Why is this commented out? Seems unreleated to the unejecting. If they are unused imports then they should be removed, not commented.

Reply:
unused. I am gonna remove them